content/coronavirus_landing_page.yml

```diff
@@ -18,6 +18,9 @@ content:
       link_nowrap_text: "cannot do"
   announcements_label: Announcements
   announcements:
+    - text: "Chancellor extends furlough scheme until October"
+      href: /government/news/chancellor-extends-furlough-scheme-until-october
+      published_text: Published 11 May 2020
```

Comment:
Looks good. Only thing I wondered is when you go to that URL it says published 12 May

Reply:
Oops, copy paste error! Thanks for picking that up, I'll fix it.

Reply:
👍